src/main/plugins/org.dita.pdf2/xsl/fo/troubleshooting-elements.xsl

```diff
@@ -0,0 +1,205 @@
+<?xml version='1.0'?>
+
+<!-- 20170503 SCH: Add support for troubleshooting elements. -->
+
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+    xmlns:fo="http://www.w3.org/1999/XSL/Format"
+    xmlns:dita2xslfo="http://dita-ot.sourceforge.net/ns/200910/dita2xslfo"
+    xmlns:opentopic="http://www.idiominc.com/opentopic"
+    xmlns:opentopic-index="http://www.idiominc.com/opentopic/index"
+    exclude-result-prefixes="opentopic opentopic-index dita2xslfo"
+    version="2.0">
+
+    <!-- Determines whether to generate titles for task sections. Values are YES and NO. -->
+    <xsl:param name="GENERATE-TASK-LABELS">

```

Comment:
I'm not sure we have a definite design here.

Most of our generated text Just Appears -- if you don't want it (or want it to show up differently), you have to customize. Honestly - the reason we have the strange situation with task labels is that originally we didn't have the translations, so didn't generate anything. A lot of people put their own titles in. So when we got translations and added support, there had to be a fairly easy way to turn it off, or else people expecting the old behavior might be broken. (I've seen thousands of topics that still have pseudo-titles in those sections from the early years, and still turn off task headings because of it.)

With this essentially being the first support for troubleshooting, I would probably just generate the headings, without adding a parameter to turn them off.